img/k/koho.ca.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" xml:space="preserve" id="Layer_1" x="0" y="0" style="enable-background:new 0 0 163.2 48" version="1.1" viewBox="0 0 163.2 48"><style>.st0,.st1{fill-rule:evenodd;clip-rule:evenodd;fill:#2d5ba8}.st1{fill:#ff8571}</style><path d="M0 48h60.9V0H0v48z" class="st0"/><path d="M60.9 48h102.3V0H60.9v48z" class="st1"/><path d="M104.2 22H93.8V12h-5.7v24h5.7v-8.8h10.4V36h5.7V12h-5.7v10zm34.4 8.4c-3.9 0-7.1-3.1-7.1-6.9s3.2-7 7.1-7c3.9 0 7 3.1 7 7s-3.2 6.9-7 6.9m0-19.3c-7 0-12.6 5.6-12.6 12.4s5.6 12.4 12.6 12.4c6.9 0 12.5-5.5 12.5-12.4s-5.6-12.4-12.5-12.4" class="st0"/><path d="M35.1 11.7h-7.3L18 21.1v-9.4h-5.7v23.5H18v-6.6l2.4-2.5 7.2 9.1h8.1L24.4 22l10.7-10.3zm25.3 18.7c-4 0-7.2-3.1-7.2-6.9 0-3.8 3.2-7 7.1-7h.1v-5.4h-.1c-7 0-12.6 5.6-12.6 12.4s5.6 12.4 12.6 12.4h.1v-5.5z" class="st1"/><path d="M60.4 11.1v5.4c3.9 0 7 3.1 7 7s-3.2 6.9-7 6.9v5.4c6.9 0 12.5-5.5 12.5-12.4s-5.6-12.3-12.5-12.3" class="st0"/></svg>
```

Comment:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 163.2 48"><path fill="#2d5ba8" d="M0 48h60.9V0H0v48z"/><path fill="#ff8571" d="M60.9 48h102.3V0H60.9v48z"/><path fill="#2d5ba8" d="M104.2 22H93.8V12h-5.7v24h5.7v-8.8h10.4V36h5.7V12h-5.7v10zm34.4 8.4c-3.9 0-7.1-3.1-7.1-6.9s3.2-7 7.1-7c3.9 0 7 3.1 7 7s-3.2 6.9-7 6.9m0-19.3c-7 0-12.6 5.6-12.6 12.4s5.6 12.4 12.6 12.4c6.9 0 12.5-5.5 12.5-12.4s-5.6-12.4-12.5-12.4"/><path fill="#ff8571" d="M35.1 11.7h-7.3L18 21.1v-9.4h-5.7v23.5H18v-6.6l2.4-2.5 7.2 9.1h8.1L24.4 22l10.7-10.3zm25.3 18.7c-4 0-7.2-3.1-7.2-6.9 0-3.8 3.2-7 7.1-7h.1v-5.4h-.1c-7 0-12.6 5.6-12.6 12.4s5.6 12.4 12.6 12.4h.1v-5.5z"/><path fill="#2d5ba8" d="M60.4 11.1v5.4c3.9 0 7 3.1 7 7s-3.2 6.9-7 6.9v5.4c6.9 0 12.5-5.5 12.5-12.4s-5.6-12.3-12.5-12.3"/></svg>
```

Reply:
I've tried searching online, but there is a lack of documentation for SMS 2FA when logging in. There only seems to be documentation of 2FA for [debit card loading](https://help.koho.ca/en/articles/6178806-debit-card-loading), where your bank sends you an SMS and you are prompted to input the code into Koho

Also, Koho only uses SMS 2FA if you are logging in from a new device, and once you've inputted the code that Koho has send to your phone, you no longer need to input a code every time. In spite of the lack of documentation, I believe that it's considered 2FA

![Screenshot_20240121-220604_KOHO](https://github.com/2factorauth/twofactorauth/assets/47160603/9db7c3e5-26b9-448a-84cd-b6a498959a19)

Reply:
I don't think that can be considered 2FA. I recommend changing Koho's entry to one without 2FA support.